src/lib/dlt_user.c

```diff
@@ -1308,6 +1306,15 @@ DltReturnValue dlt_register_app(const char *apid, const char *description)
 
 #ifdef DLT_TRACE_LOAD_CTRL_ENABLE
     strncpy(trace_load_settings[0].apid, dlt_user.appID, DLT_ID_SIZE);
+    if (!trace_load_context.contextID[0])
```

Comment:
It is make sense to register special context inside dlt_register_app. Because dlt_register_app api serve for register app purpose.

Reply:
It is required, as logging without calling the register api first is technically possible and is happening within our project and this change is implemented in a way that is fully compliant with existing code